tests/precompile/solidity/suites.go

```diff
@@ -6,13 +6,70 @@ package solidity
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
 	"time"
 
+	"github.com/ava-labs/avalanchego/api/health"
 	"github.com/ava-labs/subnet-evm/tests/utils"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/go-cmd/cmd"
 	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
 )
 
-var _ = ginkgo.Describe("[Precompiles]", ginkgo.Ordered, func() {
+var subnets map[string]string
+var startCmd *cmd.Cmd
+
+var _ = ginkgo.SynchronizedAfterSuite(func() {}, func() {
+	gomega.Expect(startCmd).ShouldNot(gomega.BeNil())
+	gomega.Expect(startCmd.Stop()).Should(gomega.BeNil())
+})
+
+var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
```

Comment:
could we add a comment explaining why we are using `SynchronizedBeforeSuite` here